shared/base/view.js

```diff
@@ -258,6 +258,15 @@ module.exports = BaseView = Backbone.View.extend({
       };
     }
 
+    // Allow ability to just pass the full "spec" to a lazy loaded view
+    if (this.options.fetch_spec) {
```

Comment:
It's because HTML is case insensitive, so when everything is converted to HTML on the sever and is passing data to the client-side there isn't a guarantee it will keep the casing. For example, chrome will interpret all html as lowercase only.

Ps, no worries about asking questions!  I'm really stoked about your (and everyone's) involvement with rendr!